### PR TITLE
chore: debug info

### PR DESCRIPTION
**Pull request by NGA-TRAN**
More metrics will be done in https://github.com/influxdata/idpe/issues/17087 but this is needed info for analysis the next few days

### CODE REVIEW

File: compactor2/src/driver.rs [hunk 1]
```diff
@@ -326,6 +327,14 @@ fn buil_compaction_plan(
         .upgrade_split
         .apply(files_to_compact, target_level);
 
+    info!(
```
Comment (crepererum) [inline compactor2/src/driver.rs:330]: Isn't this the same as https://github.com/influxdata/influxdb_iox/blob/main/compactor2/src/components/commit/logging.rs ?

Comment (NGA-TRAN) [reply compactor2/src/driver.rs:330]: No, that one missed `files_to_keep` which includes non-overlapping files that I really want to see how many files we do not recompact

Comment (NGA-TRAN) [reply compactor2/src/driver.rs:330]: This info needed before commit

File: compactor2/src/driver.rs [hunk 2]
```diff
@@ -326,6 +327,14 @@ fn buil_compaction_plan(
...
+        target_level = target_level.to_string(),
+        files_to_compacts = files_to_compact.len(),
+        files_to_upgrade = files_to_upgrade.len(),
+        files_to_keps = files_to_keep.len(),
```
Comment (NGA-TRAN) [inline compactor2/src/driver.rs:334]:
```suggestion
        files_to_keep = files_to_keep.len(),
```

Comment (NGA-TRAN) [reply compactor2/src/driver.rs:334]: The files_to_keep is critical for me to analyze how well `TargetLevel` version works

**Review by alamb — APPROVED** [review]

**Pull request merged**